main.nf

```diff
@@ -328,7 +328,7 @@ if (params.bt2n != 0 && params.bt2n != 1) {
 }
 
 // Index files provided? Then check whether they are correct and complete
-if( params.bwa_index != '' && (params.mapper == 'bwaaln' | params.mapper == 'bwamem')){
+if( params.bwa_index != '' && (params.mapper == 'bwaaln' | params.mapper == 'bwamem' | params.mapper == 'circularmapper')){
```

Comment:
We're going to be able to remove so much of this code as soon as the schema parameter validation stuff is written 😍